projects/vdk-core/src/vdk/internal/builtin_plugins/ingestion/ingester_router.py

```diff
@@ -226,10 +265,44 @@ def __initialize_ingester(self, method) -> IngesterBase:
                 self._state.get(CommonStoreKeys.OP_ID),
                 ingester_plugin,
                 IngesterConfiguration(config=self._cfg),
+                pre_processors=self._initialized_pre_processors,
+                post_processors=self._initialized_post_processors,
             )
 
         return self._cached_ingesters[method]
 
+    def __get_plugin_sequence(self, key: str) -> List:
```

Comment:
seems like a config utility method, for parsing configuration sequences by delimiter;

Reply:
Indeed. Moved to utils.